keyboards/helixlab/aaru65/fader.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 luantty2 (@luantty2)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#include QMK_KEYBOARD_H
+#ifdef VIA_ENABLE
+#    include "via.h"
+#endif
+
+#define FADER_SAMPLE_RATE 10
+#define ADC_DEADBAND 255
+#define ADC_ERROR 0xFF
+
+#ifdef VIA_ENABLE
+#    define EECONFIG_FADER ((uint32_t*)VIA_EEPROM_CUSTOM_CONFIG_ADDR)
+#else
+#    define EECONFIG_FADER ((uint32_t*)EECONFIG_SIZE)
+#endif
```

Comment:
```suggestion
```

the keyboard address can be used here instead, and is set to a 32bit size by default. 

https://docs.qmk.fm/#/feature_eeprom

Reply:
The kb section of eeprom has been used by `soldered/rgb_indicator.c`, have to allocate a new address for the fader.